components/Buttons/src/MDCFloatingButton.m

```diff
@@ -23,14 +23,14 @@
 
 static const CGFloat MDCFloatingButtonDefaultDimension = 56;
 static const CGFloat MDCFloatingButtonMiniDimension = 40;
-static const CGFloat MDCFloatingButtonDefaultImageTitleSpacing = 8;
+static const CGFloat MDCFloatingButtonDefaultImageTitleSpace = 8;
 static const UIEdgeInsets internalLayoutSpacingInsets = (UIEdgeInsets){0, 16, 0, 24};
```

Comment:
internalLayoutInsets. "Spacing" provides no new information. Its an inset.

Reply:
Well, they aren't actually used as insets - I'm just reusing that data structure. If it makes more sense, I can keep the values in a non-insets struct.

Reply:
Changed in #2578